carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/decorator/ExtendedFieldDecorator.java

```diff
@@ -77,6 +79,10 @@ public Object decorate(ClassLoader loader, Field field) {
         if (locator == null) {
             return null;
         }
+		if (((ExtendedElementLocatorFactory) factory).isRootElementUsed()) {
+			LOGGER.debug("Setting setShouldCache=false for locator: " + getLocatorBy(locator).toString());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Setting setShouldCache=false for locator: " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.207.202/sonarqube/coding_rules#rule_key=squid%3AS1192)